### DISCUSSION TITLE
Change title slide text size and color in revealjs

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#5687] [Q&A] [answered]
How can I change title slide text size and color in revealjs?

**Comment by mcanouil** [comment, marked as answer]
Use (S)CSS, see <https://quarto.org/docs/presentations/revealjs/themes.html>.

To target the right element use your browser developper mode, usually right mouse button and "inspect".

````qmd
---
title: "Untitled"
format: revealjs
include-in-header:
  - text: |
      <style>
      #title-slide .title {
        font-size: 4.5em;
        color: #b22222;
      }
      </style>
---

````

**Comment by ManuelSpinola** [comment]
Thank you very much.

El jue, 25 may 2023 a las 11:44, Mickaël Canouil ***@***.***>)
escribió:

> Use (S)CSS, see https://quarto.org/docs/presentations/revealjs/themes.html
> .
>
> To target the right element use your browser developper mode, usually
> right mouse button and "inspect".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/5687#discussioncomment-6003673>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBY2EYG745FL4X3FGYLXH6K7XANCNFSM6AAAAAAYPFCGKQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app
